packages/babel-plugin-transform-class-properties/test/fixtures/nested-class/super-property-in-decorator/output.js

```diff
@@ -24,7 +24,7 @@ let Outer = /*#__PURE__*/function (_Hello) {
       _init_extra_hello(this);
     });
     _Inner = Inner;
-    [_init_hello, _init_extra_hello] = babelHelpers.applyDecs2311(_Inner, [], [[babelHelpers.get((_thisSuper = babelHelpers.assertThisInitialized(_this), babelHelpers.getPrototypeOf(Outer.prototype)), "dec", _thisSuper), 0, "hello"]]).e;
+    [_init_hello, _init_extra_hello] = babelHelpers.applyDecs2311(_Inner, [], [[babelHelpers.get((_thisSuper = _this, babelHelpers.getPrototypeOf(Outer.prototype)), "dec", _thisSuper), 0, "hello"]]).e;
```

Comment:
It would be great to avoid the `_thisSuper` variable and just use `_this`, but I'm not sure if that's actually possible.

Reply:
The `thisSuper` was introduced in https://github.com/babel/babel/pull/11480, to ensure the this binding is resolved before the computed property is evaluated. As long as we have called `_assertThisInitialized` on the `this` expression before, it seems okay to remove the `thisSuper`.